phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/LocalRepositoryDetailsPanel.java

```diff
@@ -43,7 +43,7 @@ public LocalRepositoryDetailsPanel() {
     private void populate() {
         this.pathField = new TextField();
 
-        this.openBrowser = new Button(tr("Open directory chooser"));
+        this.openBrowser = new Button(tr("Choose Directory…"));
```

Comment:
See above.

Reply:
Perhaps "Open Directory" is preferable for consistency:
https://transvision.mozfr.org/?recherche=Open+Directory&repo=gecko_strings&sourcelocale=en-US&locale=fr&search_type=strings
?

Reply:
No. The dialog does not open a directory. It allows you to choose a directory where the repository is located.

Reply:
...